content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -95,6 +95,7 @@ kubectl apply -f ./my1.yaml -f ./my2.yaml      # create from multiple files
 kubectl apply -f ./dir                         # create resource(s) in all manifest files in dir
 kubectl apply -f https://git.io/vPieo          # create resource(s) from url
 kubectl create deployment nginx --image=nginx  # start a single instance of nginx
+kubectl run --generator=run-pod/v1 nginx --image=nginx # create nginx pod without creating deployment
```

Comment:
```suggestion
kubectl run --generator=run-pod/v1 nginx --image=nginx # create nginx pod without creating a Deployment
```

@jayeshmahajan you should also indent the comments on lines 93 to 99 to match the new line you're adding

Reply:
Hello @jayeshmahajan .
In the kubectl cheatsheet, the comments use lowercase for resource names (deployment vs Deployment). The comment seems okay.
As @sftim mentioned, typically, when a new command is added, the comments are lined up for that section.
Would you have time to align the comments for the other commands to match your addition?
Thanks!